jobs/build/refresh-images/Jenkinsfile

```diff
@@ -52,24 +52,92 @@ node('openshift-build-1') {
-                              [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'jupierce@redhat.com,ahaile@redhat.com,smunilla@redhat.com,bbarcaro@redhat.com', description: 'Success Mailing List', name: 'MAIL_LIST_SUCCESS'],
-                              [$class: 'hudson.model.StringParameterDefinition', defaultValue: 'jupierce@redhat.com,ahaile@redhat.com,smunilla@redhat.com,bbarcaro@redhat.com,mlamouri@redhat.com', description: 'Failure Mailing List', name: 'MAIL_LIST_FAILURE'],
-                              [$class: 'BooleanParameterDefinition', defaultValue: false, description: 'Mock run to pickup new Jenkins parameters?.', name: 'MOCK'],
-                              // TODO reenable when the mirrors have the necessary puddles
-                              [$class: 'hudson.model.BooleanParameterDefinition', defaultValue: false, description: 'Build golden image after building images?', name: 'BUILD_AMI'],
-                              [$class: 'hudson.model.StringParameterDefinition', defaultValue: "", description: 'Exclude these images from builds. Comma or space separated list. (i.e cri-o-docker,aos3-installation-docker)', name: 'BUILD_EXCLUSIONS'],
-                              [$class: 'hudson.model.StringParameterDefinition', defaultValue: '', description: 'Advisory Number to attach new images to', name: 'ADVISORY_ID'],
-                      ]
-             ]]
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '720'
+                )
+            ),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'GITHUB_BASE',
+                        description: 'Github base for repos',
+                        $class: 'hudson.model.ChoiceParameterDefinition',
+                        choices: "git@github.com:openshift\ngit@github.com:jupierce\ngit@github.com:jupierce-aos-cd-bot\ngit@github.com:adammhaile-aos-cd-bot",
+                        defaultValue: 'git@github.com:openshift'
+                    ],
+                    [
+                        name: 'OSE_MAJOR',
+                        description: 'OSE Major Version',
+                        $class: 'hudson.model.ChoiceParameterDefinition',
+                        choices: "3",
+                        defaultValue: '3'
+                    ],
+                    [
+                        name: 'OSE_MINOR',
+                        description: 'OSE Minor Version',
+                        $class: 'hudson.model.ChoiceParameterDefinition',
+                        choices: "1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15",
```

Comment:
Using join('\n') model for choices parameters is fine here. Parameters are added/modified infrequently and testing them is relatively fast. Up to you whether you want to take that on in this refactor.

Reply:
I looked at joining the numbers or generating a range. IN the case of "1..13" or what ever it ended up being less clear than the string list. If it gets much longer it might make more sense.

Reply:
Ultimately, the versions themselves will be in a pipeline library (sourced from a shared location). We can dial in the extent of the refactor to make here:
1. Leave it as-is.
2. Use join as a quick improvement.
3. Refactor to use BUILD_VERSION (combo of major.minor like build/ocp job) and join('\n') to reduce potential for human error. 

Generating a sequence will be problematic if we ever move to a 4.0, so hoping to be at shared library at that point.

Reply:
I'm for leaving this alone right now.  It's out of scope and not hurting anything.
Re: if ever... deal with it then.  It's just a string.
Join isn't an improvement in this case, the string reads cleaner.